geode-dunit/src/main/java/org/apache/geode/cache30/RegionTestCase.java

```diff
@@ -3736,7 +3736,7 @@ private void remoteTestPostSnapshot(String name, boolean isController, boolean i
 
     boolean expectData =
         isController || postSnapshotRegion.getAttributes().getMirrorType().isMirrored()
```

Comment:
Can `getMirrorType().isMirrored()` be replaced with information from the `DataPolicy`?

Reply:
Don't know. It is out of scope for this current change.